### PR TITLE
Add layer analytics bundle

### PR DESCRIPTION
**Pull request by kasparia**

### CODE REVIEW

File: bundles/framework/admin-layeranalytics/LayerAnalyticsContent.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        />
+    );
+};
+
+LayerAnalyticsContent.propTypes = {
+    analyticsData: PropTypes.array.isRequired
+};
+
+const contextWrap = LocaleConsumer(LayerAnalyticsContent);
+export { contextWrap as LayerAnalyticsContent };
```
Comment (ZakarFin) [inline bundles/framework/admin-layeranalytics/LayerAnalyticsContent.jsx:45]: This should work without the LocaleConsumer wrapping as we are not using the getMessage prop it provides. Try removing it and export LayerAnalyticsContent as is.

File: bundles/framework/admin-layeranalytics/resources/locale/en.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Oskari.registerLocalization(
+    {
+        "lang": "fi",
```
Comment (ZakarFin) [inline bundles/framework/admin-layeranalytics/resources/locale/en.js:3]:
```suggestion
        "lang": "en",
```

File: bundles/framework/admin-layeranalytics/resources/locale/sv.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Oskari.registerLocalization(
+    {
+        "lang": "fi",
```
Comment (ZakarFin) [inline bundles/framework/admin-layeranalytics/resources/locale/sv.js:3]:
```suggestion
        "lang": "sv",
```

File: bundles/framework/admin-layeranalytics/LayerAnalyticsContent.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import { Message } from 'oskari-ui';
+
+const columnSettings = [
+    {
+        align: 'left',
+        title: <Message messageKey='flyout.idTitle' />,
+        dataIndex: 'id',
+        key: 'id',
+        // eslint-disable-this-line react/display-name
+        render: text => <a>{text}</a>
```
Comment (ZakarFin) [inline bundles/framework/admin-layeranalytics/LayerAnalyticsContent.jsx:14]: Is there a reason for wrapping this in `<a>`?

File: bundles/framework/admin-layeranalytics/Tile.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
+/**
+ * @class Oskari.framework.bundle.admin-layeranalytics.Tile
+ * Renders the "layer analytics admin" tile.
+ */
+Oskari.clazz.define('Oskari.framework.bundle.admin-layeranalytics.Tile',
```
Comment (ZakarFin) [inline bundles/framework/admin-layeranalytics/Tile.js:5]: Nothing particularly wrong here but it's a lot of code just to get a "tile" without any customization. We have a DefaultTile under framework/divmanazer/extension that could help with this or you could extend DefaultExtension with instance.js. That would make this file unnecessary. It's ok for now but I really wish we could do this much simpler.

File: bundles/framework/admin-layeranalytics/Flyout.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            );
+        },
+        startPlugin: function () {},
+        setSpinnerState: function (spinnerState) {
+            if (!spinnerState) {
+                this.progressSpinner.stop();
+            } else {
+                this.progressSpinner.start();
+            }
+        }
```
Comment (ZakarFin) [inline bundles/framework/admin-layeranalytics/Flyout.js:56]: This should be replaced with `<Spin>` from AntD. Perhaps making "loading" as a parameter for updateListing()?

**Comment by ZakarFin** [comment]
As a general note this is an admin bundle so we should move it from under `framework` to under `admin`. But more importantly as this is application specific functionality for now we should move it to https://github.com/nls-oskari/pti-frontend/tree/master/bundles/paikkatietoikkuna :)

**Comment by kasparia** [comment]
Moving this to pti-frontend repo now and closing this.

**Pull request closed without merging**